resources/core/charts/kubeless/templates/tests/test-kubeless.yaml

```diff
@@ -73,6 +73,7 @@ subjects:
   name: test-{{ template "fullname" . }}
   namespace: {{ .Release.Namespace }}
 ---
+{{- if not .Values.global.isLocalEnv }}
```

Comment:
should we skip the complete file ?  We can skip the deployment of cluster role too ?

Reply:
We can skip the deployment of `clusterrole` for now but we will introduce it again in this [PR](https://github.com/kyma-project/kyma/pull/1754) as creation/deletion of namespace will be done.  This PR is targetted only to have aesthetic changes and keep the existing functionality as is.